accord-core/src/main/java/accord/impl/InMemoryCommandStore.java

```diff
@@ -459,45 +408,63 @@ void update(Ranges add)
         }
     }
 
-    class CFKLoader implements CommandLoader<TxnId>
+    static class CFKEntry extends TxnId
```

Comment:
ill split this into a different PR once evict is in, but to make sure we actually test the cache-miss case I added a LRU~ish "cache" and dropped this class as its broken... we are pulling a globally modifiable reference within an immutable object (CommandsForKeys)?